apps/nrm/index.js

```diff
@@ -8,6 +8,30 @@ const dataToPdf = require('./behaviours/data-to-pdf');
 const generateSendPdf = require('./behaviours/generate-send-pdf');
 const saveMissingData = require('./behaviours/save-missing-data');
 const transferMissingData = require('./behaviours/transfer-missing-data');
+const pvUnderAge = require('./behaviours/pv-under-age');
+const pvUnderAgeAtTimeOfExploitation = require('./behaviours/pv-under-age-at-time-of-exploitation');
+const localAuthorityContactedAboutChild = require('./behaviours/local-authority-contacted-about-child');
+const coOperateWithPolice = require('./behaviours/co-operate-with-police');
+const confirm = require('./behaviours/confirm');
+const pvName = require('./behaviours/pv-name');
+const pvNationality = require('./behaviours/pv-nationality');
+const pvContactDetails = require('./behaviours/pv-contact-details');
+const frLocation = require('./behaviours/fr-location.js');
+const whatHappend = require('./behaviours/what-happend');
+const pvGender = require('./behaviours/pv-gender');
+const pvPhoneNumber = require('./behaviours/pv-phone-number');
+const frDetails = require('./behaviours/fr-details');
+const doesPvHaveChildren = require('./behaviours/does-pv-have-children');
+const reportedToPolice = require('./behaviours/reported-to-police');
+const pvDob = require('./behaviours/pv-dob');
+const pvHoReference = require('./behaviours/pv-ho-reference');
+const pvWantToSubmitNrm = require('./behaviours/pv-want-to-submit-nrm');
+const doesPvNeedSupport = require('./behaviours/does-pv-need-support');
+const pvInterpreterRequirements = require('./behaviours/pv-interpreter-requirements');
+const refuseNrm = require('./behaviours/refuse-nrm');
+const whoContact = require('./behaviours/who-contact');
+const someoneElse = require('./behaviours/someone-else');
+const frAlternateContact = require('./behaviours/fr-alternate-contact');
```

Comment:
Wouldn’t it be more clear if the behaviours were prefixed with backLink

Reply:
Some behaviours  like pv-gender.js have amendments to the backlink and next step functions and others such as someone-else.js only the backlink was amended.

Should I prefix all the new behaviours with backLink nevertheless? @sulthan-ahmed

Reply:
for the backlink ones prefix with bkLinkWhoContact
for both bkLinkNextSomeoneElse

Reply:
Thanks